src/utils/constants.ts

```diff
@@ -75,8 +75,8 @@ export const ARWEAVE_TX_LENGTH = 43;
 export const EMAIL_REGEX = new RegExp(
   "([!#-'*+/-9=?A-Z^-~-]+(.[!#-'*+/-9=?A-Z^-~-]+)*|\"([]!#-[^-~ \t]|(\\[\t -~]))+\")@([!#-'*+/-9=?A-Z^-~-]+(.[!#-'*+/-9=?A-Z^-~-]+)*|[[\t -Z^-~]*])", // eslint-disable-line
 );
-export const TTL_SECONDS_REGEX = new RegExp('^[0-9]{3,7}$');
-export const TTL_SECONDS_ENTRY_REGEX = new RegExp('^[0-9]{1,7}$');
+export const TTL_SECONDS_REGEX = new RegExp('^[0-9]{2,5}$');
+export const TTL_SECONDS_ENTRY_REGEX = new RegExp('^[0-9]{1,5}$');
```

Comment:
odd we have regex for known numbers

Reply:
Yeah i think this is redundant, will see if possible to remove

Reply:
[1379e15](https://github.com/ar-io/arns-react/pull/661/commits/1379e15f0d520761fa9a794df073648b3d603dcd)